plaso/parsers/firefox_cache.py

```diff
@@ -433,7 +433,7 @@ def _ValidateCacheFileMetadataHeader(self, cache_file_metadata_header):
     return (
         cache_file_metadata_header.key_size > 0 and
         cache_file_metadata_header.key_size < self._MAXIMUM_URL_LENGTH and
-        cache_file_metadata_header.format_version == 1 and
+        cache_file_metadata_header.format_version in {1, 2, 3} and
```

Comment:
For codebase consistency, maybe use a tuple instead `(1, 2, 3)` instead of a set